### ISSUE TITLE
Linux setup docs missing step to install curses development package for Fedora

### ISSUE THREAD

**Issue by nashif** [zephyriot/zep-jira14#425]
**_Reported by Henry Bruce:_**

I followed instructions on https://www.zephyrproject.org/doc/getting_started/installation_linux.html to set up for development on my Fedora 23 workstation.

When I ran "make menuconfig" for hello_world nanokernel sample it failed as follows

```
$ make menuconfig
make[1]: Entering directory '/path/to/zephyr'
make[2]: Entering directory 'path/to/zephyr/samples/hello_word/nanokernel/outdir'
  GEN     ./Makefile
  HOSTCC  scripts/kconfig/mconf.o
In file included from /path/to/zephyr/scripts/kconfig/mconf.c:23:0:
/path/to/zephyr/scripts/kconfig/lxdialog/dialog.h:38:20: fatal error: curses.h: No such file or directory
compilation terminated.
```

Build was fixed with "sudo dnf install ncurses-devel"

(Imported from Jira ZEP-478)

**Comment by nashif** [comment]
**_by David Kinder:_**

I can add ```ncurses-devel``` to the required packages for Fedora.  Should I also add it for Ubuntu?
Or is this not really required for core zephyr development, and we should add documentation around the samples that says what additional packages are needed to work with them?

**Comment by nashif** [comment]
**_by Inaky Perez-Gonzalez:_**

I think it can be safely added to both

Because for menuconfig it will be needed.

I am not sure ncurses-devel is the right name in Ubuntu; probably ncurses-dev, but it'd be have to verified. I don't have an Ubuntu machine.

**Comment by nashif** [comment]
**_by David Kinder:_**

it's ```sudo apt-get install ncurses-dev``` on my ubuntu system.  Doc updated with https://gerrit.zephyrproject.org/r/#/c/3902/

**Comment by nashif** [comment]
**_by Mark Linkmeyer:_**

_Henry Bruce_ , as the Reporter of this bug we need you to verify it's fixed since it's in the Resolved state.  Please verify it ASAP so we know whether or not more work is required to fix it before the upcoming 1.5 release.  If it's fixed, please move it to the Verified state.  Thanks!

**Comment by nashif** [comment]
**_by Mark Linkmeyer:_**

Fixing incorrect priority
